node_lambnik/src/tiler/package.json

```diff
@@ -43,6 +44,8 @@
     "nodemon": "^1.17.5"
   },
   "dependencies": {
+    "@mapbox/sphericalmercator": "^1.1.0",
+    "babel-runtime": "6",
```

Comment:
Am I remembering correctly that you weren't sure if this was necessary?  Or am I confusing this with a conversation about something else?  In any case, where does this come into play?

Reply:
I got SphericalMercator working, turns out I was just telling it to use the wrong projection. Re: babel-runtime, I ran into an issue with getting Promises to run reliably after transpilation, so I added that at the internet's suggestion and it solved whatever issue that was.